app/views/play/modal/PollModal.coffee

```diff
@@ -77,7 +77,26 @@ module.exports = class PollModal extends ModalView
     pollVotes[@poll.id] = $selectedAnswer.data('answer').toString()
     @userPollsRecord.set 'polls', pollVotes
     @updateAnswers true
-    @userPollsRecord.save {polls: pollVotes}, {success: => @awardRandomGems?()}
+    @userPollsRecord.save {polls: pollVotes}, {success: => 
+      @awardRandomGems?()
+
+      myAnswer = (@userPollsRecord.get('polls') ? {})[@poll.id]
+      answerObj = _.find(@poll.get('answers'), (answer) => answer.key == myAnswer) or {}
+      nextPollId = answerObj.nextPoll
+      
+      # The following block allows for the user to be indecisive with their answer, updating UI accordingly.
```

Comment:
Curious about the comment - how does this allow the user to be indecisive?

Reply:
A user can change their answer and move their vote. This code ensures that the button changes between "Done" and "Next" depending on whether their selected answer has a `nextPoll` property.

![indecisive_clicks](https://user-images.githubusercontent.com/15080861/78064214-41f98d00-7346-11ea-9ddb-f1a290cca08e.gif)

Reply:
Got it, thanks for the gif :)